### PR TITLE
Devise

### PR DESCRIPTION
**Pull request by MamoonaT**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,4 +2,20 @@
 
 # ApplicationController
 class ApplicationController < ActionController::Base
+  # before_action :update_allowed_parameters, if: :devise_controller?
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  private
+
+  def configure_permitted_parameters
```
Comment (gullariz12) [inline app/controllers/application_controller.rb]: Can we permit these parameters in the devise registration controller?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -2,4 +2,20 @@
...
+    added_attrs = %i[username phone_number email password password_confirmation remember_me]
+    devise_parameter_sanitizer.permit :sign_up, keys: added_attrs
+    devise_parameter_sanitizer.permit :account_update, keys: added_attrs
+  end
+
+  # def update_allowed_parameters
```
Comment (gullariz12) [inline app/controllers/application_controller.rb]: Can we remove this commented code if it not required?

**Review by gullariz12 — CHANGES_REQUESTED** [review]

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (MamoonaT) [reply app/controllers/application_controller.rb]: Removed

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (MamoonaT) [reply app/controllers/application_controller.rb]: Done

**Review by gullariz12 — APPROVED** [review]

**Pull request merged**